--- /mnt/stuff/xhome/dawg/aos/el-build-home/cache/build.ascendos.org___linux__ascendos__SRPMS.repo-diff/nss-3.12.7-2.tuv/./nss.spec	2011-10-28 07:55:03.000000000 +0000
+++ /mnt/stuff/xhome/dawg/aos/el-build-home/cache/build.ascendos.org___linux__ascendos__SRPMS.repo-diff/nss-3.12.7-2.tdv/./nss.spec	2011-10-28 07:55:03.000000000 +0000
@@ -7,7 +7,7 @@
 Summary:          Network Security Services
 Name:             nss
 Version:          3.12.7
-Release:          2%{?dist}
+Release:          2%{?dist}.0.asc
 License:          MPLv1.1 or GPLv2+ or LGPLv2+
 URL:              http://www.mozilla.org/projects/security/pki/nss/
 Group:            System Environment/Libraries
@@ -245,6 +245,10 @@
 # disable the ssl test suites for faster test builds
 #%%global nss_ssl_tests "normal_fips"
 #%%%global nss_ssl_run "cov auth"
+%global nss_cycles " "
+%global nss_tests " "
+%global nss_ssl_tests " "
+%global nss_ssl_run " "
 
 HOST=localhost DOMSUF=localdomain PORT=$MYRAND NSS_CYCLES=%{?nss_cycles} NSS_TESTS=%{?nss_tests} NSS_SSL_TESTS=%{?nss_ssl_tests} NSS_SSL_RUN=%{?nss_ssl_run} ./all.sh
 
@@ -486,6 +490,9 @@
 
 
 %changelog
+* Mon Jul 25 2011 Troy Dawson <dawson@ascendos.org> - 3.12.7-2.0.asc
+- skipped tests that the comments say are safe to skip
+
 * Thu Aug 27 2010 Kai Engert <kengert@redhat.com> - 3.12.7-2
 - Increase release version number, no code changes
 
